SortApp/SortApp.BL.Test/Sorting/BubbleSortTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SortApp.BL.Test.Sorting
+{
+	/// <summary>
+	/// Represents the tests executed towards the <see cref="SortApp.BL.Sortings.BubbleSort{T}"/> sorting class.
+	/// </summary>
+	/// <owner>Oleh Petrenko</owner>
+	[TestClass]
+	public sealed class BubbleSortTests
```

Comment:
There should be a test for case when passed array is null (in all tests). Usually when it happens - we throw new ArgumentNullException